openmdao/utils/jax_utils.py

```diff
@@ -0,0 +1,70 @@
+"""
+Utilities for the use of jax in combination with OpenMDAO.
+"""
+from collections.abc import Callable
+
+try:
+    import jax
+except ImportError:
+    jax = None
+
+
+def jit_stub(f):
```

Comment:
Do you want to add **kwargs to that function signature? So when you do, for example,

```
    @partial(jit, static_argnums=(0,))
```
there are no issues when jit is jit_stub.